TSQLLint.Console/ConfigHandler/CommandLineOptionHandler.cs

```diff
@@ -56,47 +56,71 @@ private bool HandleConfigOptions()
             {
                 return true;
             }
-            
-            ReportUsage();
+
+            if (!createdConfig)
+            {
+                ReportUsage();
+            }
+
             return false;
         }
 
-        private void SetupConfig()
+        private bool CheckOptionsForConfigFile()
         {
-            var useInMemoryRules = false;
+            var allowInMemoryRules = false;
             var configFile = _commandLineOptions.ConfigFile;
 
             if (string.IsNullOrWhiteSpace(configFile))
             {
                 _commandLineOptions.ConfigFile = configFile = _configFileFinder.DefaultConfigFileName;
-                useInMemoryRules = !(_commandLineOptions.Init || _commandLineOptions.Force);
+                allowInMemoryRules = !(_commandLineOptions.Init || _commandLineOptions.Force);
             }
             else
             {
                 _commandLineOptions.ConfigFile = configFile = configFile.Trim();
             }
 
-            LoadConfig(configFile, useInMemoryRules);
-        }
-
-        private void LoadConfig(string configFile, bool useInMemoryRules)
-        {
             var configFileExists = FileExists(configFile);
-            if (useInMemoryRules && !configFileExists)
+            var createdConfigFile = false;
+            if (UseInMemoryRules(configFileExists, allowInMemoryRules))
             {
-                _commandLineOptions.DefaultConfigRules = _configFileGenerator.GetDefaultConfigRules();
-                _commandLineOptions.ConfigFile = null;
+                // Did not create config file
```

Comment:
empty bock?

Reply:
So more of a formatting question. The default is already false so this option does not need to do anything (hence I put a comment to indicate what the variable meant).

The alternate format is removing the local variable and use returns which removes the need for empty block but then has a series of returns